src/components/atoms/TabLabel.tsx

```diff
@@ -4,26 +4,51 @@ import styled from 'styled-components';
 interface IProps {
   isActive: boolean;
   className?: string;
-  tabIndex: number;
-  onClick: (index: number) => void;
+  label: string;
+  onClick: (label: string) => void;
 }
 
-const LI = styled.li`
+const BUTTON = styled.button`
```

Comment:
As this is not exported, you could just call this `Button`. I have no clue why I originally called it `LI`.

Reply:
```suggestion
const Button = styled.button`
```

Reply:
aha, might have been that LI is an abbreviation of List Item.
Fixed.